source/_includes/site/sidebar.html

```diff
@@ -48,11 +48,37 @@ <h1 class="title delta">On this page</h1>
     {%- endunless -%}
   {% endif %} 
 
+  {%- if page.ha_platforms.first -%}
+  <section class="aside-module grid__item one-whole lap-one-half">
+    <div class='section'>
+      <h1 class="title delta">Platforms</h1>
+      This integration creates entities from the following platforms.
+      See these general pages for details on using their triggers and actions in automations:
+    </div>
+    <div class='section'>
+      <ul class='unstyled'>
+      {%- for platform in page.ha_platforms -%}
+        <li>
+          <a href='/integrations/{{ platform }}'>{{ platform | capitalize }}</a>
```

Comment:
There are plafroms that contain a `_` that should be replaced with a space.
So it becomes `Device tracker` instead of `Device_tracker`